docs/account/my-organization/role-management.md

```diff
@@ -19,210 +19,333 @@ tags:
-| None       | The user cannot reach any details about distribution profiles, Apple Devices, App Versions Report, and App Sharing Report.              |
-
-:::info
-
-Manager or Operator Distribution Profile permission can send to enterprise app store if user has Manager, Uploader or Operator Enterprise App Store permission.
+### Testing Distribution Permissions
+
+The following table details the roles and restrictions for the [**Testing Distribution**](/distribute) module. Please refer to the modules related information and coution notes. 
+
+| Testing Distribution | Scopes                                 | Owner | Manager | Operator | Ext. Operator | Viewer |
+|----------------------|----------------------------------------|-------|---------|----------|---------------|--------|
+| Distribution Profile | Add/Delete/Update Distribution Profile | ✅     | ✅       | ⛔        | ⛔             | ⛔      |
+| Distribution Profile | Setting Update Distribution Profile    | ✅     | ✅       | ⛔        | ⛔             | ⛔      |
+| Distribution Profile | List Distribution Profiles             | ✅     | ✅       | ✅        | ✅             | ✅      |
+| App Version          | Add/Delete/Update App Version          | ✅     | ✅       | ✅        | ✅             | ⛔      |
+| App Version Actions  | Send to Testers                        | ✅     | ✅       | ✅        | ✅             | ⛔      |
+| App Version Actions  | Send to Enterprise App Store           | ✅     | ✅       | ✅        | ⛔             | ⛔      |
+| App Version Actions  | Send to Publish                        | ✅     | ✅       | ✅        | ⛔             | ⛔      |
+| Settings             | Select Authentication Type             | ✅     | ⛔       | ⛔        | ⛔             | ⛔      |
+| Settings             | View Authentication Settings           | ✅     | ✅       | ✅        | ⛔             | ✅      |
+| Apple Device         | Add/Delete Apple Device                | ✅     | ✅       | ⛔        | ⛔             | ⛔      |
+| Apple Device         | Register Apple Device                  | ✅     | ✅       | ⛔        | ⛔             | ⛔      |
+| Apple Device         | Adding New Device to Provision         | ✅     | ✅       | ⛔        | ⛔             | ⛔      |
+| Apple Device         | Sync from Apple Developer              | ✅     | ✅       | ✅        | ⛔             | ⛔      |
+| Apple Device         | List Apple Device                      | ✅     | ✅       | ✅        | ⛔             | ✅      |
+| Report               | List Reports App Version               | ✅     | ✅       | ✅        | ⛔             | ✅      |
+| Report               | List Reports App Sharing               | ✅     | ✅       | ✅        | ⛔             | ✅      |
+
+:::caution Authentication Settings
+
+If the selected Authentication type is Static login, Manager role can change **Username** and **Password**. However, it cannot change the content for other Authentication types such as **LDAP** or **SSO**.
 
```

Comment:
Need to add a note for pointing out that manager and operator distribution roles need to have a manager signing identity role in order to sync apple device from apple developer.